src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -1228,6 +1228,7 @@ void ObjectMgr::LoadGameObjectAddons()
         gameObjectAddon.ParentRotation = QuaternionData(fields[1].GetFloat(), fields[2].GetFloat(), fields[3].GetFloat(), fields[4].GetFloat());
         gameObjectAddon.invisibilityType = InvisibilityType(fields[5].GetUInt8());
         gameObjectAddon.InvisibilityValue = fields[6].GetUInt32();
+        gameObjectAddon.worldEffectID = fields[7].GetUInt32();
```

Comment:
is there a way to validate this id?

Reply:
Maybe WorldEffect.db2
Need to investigate it.